pkg/controller/add_nodemaintenance.go

```diff
@@ -1,10 +1,10 @@
 package controller
 
 import (
-	"kubevirt.io/node-maintenance-operator/pkg/controller/nodemaintenance"
+	"kubevirt.io/node-maintenance-operator/pkg/controller/machinemaintenance"
```

Comment:
why is it so important to rename this to machinemaintenance ? (it kinds of defies the repository's name and purpose since we are operating on k8s nodes - unless the intent is more then just renaming the package here)

Reply:
renamed back into nodemaintenance